pom.xml

```diff
@@ -41,7 +41,7 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <guava.version>28.0-jre</guava.version>
     <resolver.version>1.4.0</resolver.version>
```

Comment:
It looks like this PR would downgrade our resolver version to 1.3.3. If so, I'm not sure we want to do that.

Reply:
I see maven-resolver-api 1.4.0 in its dependency tree https://gist.github.com/suztomo/f3a99fa9c285c00552c751911e342b59.

Both this pom's parent (dependency-parent) and [`org.apache.maven:maven:3.6.1`](https://search.maven.org/artifact/org.apache.maven/maven/3.6.1/pom) try to set maven-resolver-api version. Seems the former wins.

Reply:
I just created #810 to clarify maven-related dependencies should have provided scope, because these dependencies are already there when Maven runs an enforcer rule.

Reply:
Created #813

Reply:
813 merged.